content/blog/vlan-versus-vxlan.md

```diff
@@ -0,0 +1,81 @@
+---
+title: VLAN Versus VXLAN
+date: 2024-11-11T14:57:37.063Z
+author: Indira Manthri
+authorimage: /img/indira_manthri.jpg
+disable: false
+tags:
+  - aruba
+  - aruba-central
+  - networking
+---
+## Contents
+
+This technical blog gives insight on basic understanding of VLAN and VXLAN and the purpose achieved by using VLAN and VXLAN. It is interesting to compare these two features as they look similar from top level but there is lot of difference in the networking layer they are used and the way the VLAN, VXLAN headers are embedded in the packet and the scalability issues resolved by these two features. A quick look into the sample configuration provided in this blog for inquisitive networking colleagues.
```

Comment:
Suggest to slightly update the first sentence:
This technical blog post gives insight....
(FYI - a blog is a website for blog posts. You are writing a blog post).

Last sentence:
A quick look....in this blog post for inquisitive community members.

Reply:
Done